File: app/src/main/java/ru/vincetti/vimoney/data/JsonDownloader.java
package ru.vincetti.vimoney.data;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;
import ru.vincetti.vimoney.models.json.ConfigFile;

public interface JsonDownloader {

    @GET("config.json")
    Call<ConfigFile> loadPreferences(@Query("q") String locale);
}

File: app/src/main/java/ru/vincetti/vimoney/data/JsonDownloader.kt
package ru.vincetti.vimoney.data

import retrofit2.http.GET
import retrofit2.http.Query
import ru.vincetti.vimoney.data.models.json.ConfigFile

interface JsonDownloader {

    @GET("config.json")
    suspend fun loadPreferences(@Query("q") locale: String): ConfigFile
}
